### PR TITLE
docs: demo on Typography.paragraph

### PR DESCRIPTION
**Pull request by fireairforce**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
image
Typography.paragraph 的这个demo在一些屏幕宽一点的屏幕上可能显示效果没有那么好，导致自定义的隐藏没有显示出来，因此把行数调少一点，和上面那一行形成一个对照。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     None     |
| 🇨🇳 Chinese |   无      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/demo/ellipsis.md](https://github.com/fireairforce/ant-design/blob/typography-demo/components/typography/demo/ellipsis.md)

### CODE REVIEW

**Comment by fireairforce** [comment]
image
改成一行显示效果可能会好一点点。

File: components/typography/demo/ellipsis.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ ReactDOM.render(
       language for background applications, is refined by Ant UED Team.
     </Paragraph>
 
-    <Paragraph ellipsis={{ rows: 3, expandable: true, symbol: 'more' }}>
+    <Paragraph ellipsis={{ rows: 1, expandable: true, symbol: 'more' }}>
```
Comment (xrkffgg) [inline components/typography/demo/ellipsis.md:32]:
```suggestion
    <Paragraph ellipsis={{ rows: 2, expandable: true, symbol: 'more' }}>
```

2 行吧

**Comment by xrkffgg** [comment]
changelog 是 面向 用户来写的。

改 Demo 对用户没有影响，可以写 none .

**Comment by fireairforce** [comment]
好的

File: components/typography/demo/ellipsis.md (hunk 1 above)
Comment (fireairforce) [reply components/typography/demo/ellipsis.md]: done

**Pull request merged**